### ISSUE TITLE
AnimationPlayer track values overlapping

### ISSUE THREAD

**Issue by golddotasksquestions** [godotengine/godot#27244]
**Godot version:**
3.1 stable

**OS/device including version:**
Win

**Issue description:**
Track values are illegible because they overlapp each other visually, especially if the are longer or the keyframes closer together.
![callfunckey31](https://user-images.githubusercontent.com/47016402/54633989-87238780-4a81-11e9-8e8d-917962432bce.png)

Proposed solution: Hide the text of the previous value where the next keyframe starts.

**Comment by girng** [comment]
this can also happen when inserting sprite frames, the image for the keyframe will overlap other images
one solution i can think of (if not already added), is an option to toggle the keyframe visuals

**Comment by golddotasksquestions** [comment]
That's more like a regression than a solution though.

**Comment by girng** [comment]
not really, the regression is them overlapping in the first place, when that was never an issue (since 2014)

**Comment by golddotasksquestions** [comment]
Simply because they were not displayed within a track prior to 3.1. This is a new feature, surely not a bad one. It just defeats it's purpose if it is illegible.

**Comment by KoBeWi** [comment]
Still valid in 3.2.4 beta3

**Comment by aobrem** [comment]
Fixed by #96421
